lib/api/api.go

```diff
@@ -1173,6 +1174,21 @@ func (s *service) getDBIgnores(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (s *service) getIgnoresRaw(w http.ResponseWriter, r *http.Request) {
+	qs := r.URL.Query()
+
+	path := qs.Get("raw-ignore")
+	finalPath := path + "/.stignore"
+	bs, err := ioutil.ReadFile(finalPath)
```

Comment:
Let's call the query param `path` and use filepath.Join:

```suggestion
	folderPath := qs.Get("path")
	ignoresPath := filepath.Join(folderPath, ".stignore")
	bs, err := ioutil.ReadFile(ignoresPath)
```